common.xml

```diff
@@ -24,7 +26,9 @@
 	<property name="build.lib" value="${build.dir}/lib"/>
 	<property name="build.test.classes" value="${build.dir}/test-classes"/>
 	<property name="build.test.lib" value="${build.dir}/test-libs"/>
-	<property name="dist.jar" value="${build.lib}/opendcs.jar"/>		
+	<property name="dist.jar" value="${build.lib}/opendcs.jar"/>
+
+    <property name="build.release.dir" value="${build.dir}/release"/>
```

Comment:
whitespace strikes again

Reply:
there's a few more new tab count/line break oddities in here, lines 196, 216

Reply:
Cleaned up the whole file. Not even sure how/why the tabs not in there.